src/libstd/sys/unix/fs.rs

```diff
@@ -108,6 +108,27 @@ impl FilePermissions {
     pub fn mode(&self) -> raw::mode_t { self.mode }
 }
 
+#[unstable(feature = "file_types_posix", reason = "recently added API")]
+pub trait FileTypeUnix {
+    /// Add special unix types (block/char device, fifo and socket)
+    #[unstable(feature = "file_types_posix", reason = "recently added API")]
```

Comment:
You should actually be able to omit these unstable tags because `#[unstable]` inherits (from the trait)